src/Packetizer/Packetizer.c

```diff
@@ -49,8 +53,13 @@ Packetizer_packetize(Packetizer* const self,
     (void)source;
     (void)dataOffset;
 
+    uint16_t rawDestination = destination;
+#ifdef FORCE_APIDS
+    rawDestination = translate_apid_to_send(rawDestination);
```

Comment:
It is a good practice to use const whenever possible. Consider using:
#ifdef FORCE_APIDS
const unit16_t destinationApid = translate_apid_to_send(destination);
#else
const unit16_t destinationApid = destination;
#endif

Reply:
done